### PR TITLE
Fix typo in clang-tidy runs

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

These didn't get caught on my system because I have `run-clang-tidy`
installed. I tested in the container that this works now.

### Types of changes:

- [x] Bugfix
- [ ] New feature
- [ ] Refactor

### Component:

- [ ] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests and `clang-tidy`.
  For instructions on how to perform the CI checks locally refer to the [Dev
  guide on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Review by nilsvu — DISMISSED** [review]

File: tools/ClangTidyHash.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ EXIT_CODE=0
 RAN_CLANG_TIDY=no
 for run_clang_tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
 do
-    if command -v "${run-clang-tidy}" > /dev/null 2>&1; then
+    if command -v ${run_clang_tidy} > /dev/null 2>&1; then
```
Comment (wthrowe) [inline tools/ClangTidyHash.sh]: You need the quotes here.  Apparently `command -v` (with no arguments) returns true, although I don't see anything in the documentation to indicate that.

File: tools/ClangTidyAll.sh [hunk 2]
```diff
@@ -43,10 +43,10 @@ EXIT_CODE=0
 # Check for run-clang-tidy, then run-clang-tidy-LLVM_VERSION, and
 # if we can't find those, then just loop over the files one at a time.
 RAN_CLANG_TIDY=no
-for run-clang-tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
+for run_clang_tidy in run-clang-tidy run-clang-tidy.py "${RUN_CLANG_TIDY_BIN}" ;
 do
-    if command -v "${run-clang-tidy}" > /dev/null 2>&1; then
-        ${run-clang-tidy} -quiet ${THREADING_FLAG} -p ${BUILD_DIR} \
+    if command -v ${run_clang_tidy} > /dev/null 2>&1; then
```
Comment (wthrowe) [inline tools/ClangTidyAll.sh]: You need the quotes here.

File: tools/ClangTidyAll.sh [hunk 3]
```diff
@@ -43,10 +43,10 @@ EXIT_CODE=0
...
+        ${run_clang_tidy} -quiet ${THREADING_FLAG} -p ${BUILD_DIR} \
                           ${ALL_SOURCE_FILES} || EXIT_CODE=1
         RAN_CLANG_TIDY=yes
         break
```
Comment (wthrowe) [inline tools/ClangTidyAll.sh:52]: While you're here, there's a `$FILENAME` that should be a `$file` a few lines below this hunk.

**Review by wthrowe — CHANGES_REQUESTED** [review]

File: tools/ClangTidyHash.sh (hunk 1 above)
Comment (nilsdeppe) [reply tools/ClangTidyHash.sh]: Yay for documentation :P Thanks for catching this :)

File: tools/ClangTidyAll.sh (hunk 3 above)
Comment (nilsdeppe) [reply tools/ClangTidyAll.sh:52]: I switched to `FILENAME` in the three places in an attempt to make the `Hash` and `All` files be more consistent

**Review by wthrowe — APPROVED** [review]

**Review by kidder — APPROVED** [review]

**Pull request merged**